source/features/quick-repo-deletion.css

```diff
@@ -1,10 +1,14 @@
-:root .rgh-quick-repo-deletion[open] span {
+:root details.rgh-quick-repo-deletion[open] span {
 	position: relative;
 	z-index: 81;
 	transform-origin: top;
 }
 
-:root .rgh-quick-repo-deletion[open] summary::before {
+:root details.rgh-quick-repo-deletion[open] summary::before {
 	background-color: var(--color-scale-red-6);
 	opacity: 30%;
 }
+
+.rgh-quick-repo-deletion #readme div {
+	z-index: 50 !important;
```

Comment:
We can’t lower this, we risk creating more issues in an unrelated area. Can you change the zIndex on the delete fork button only when it’s open?

Reply:
<img width="1012" alt="Screen Shot 2" src="https://user-images.githubusercontent.com/1402241/115107823-59c6a200-9f97-11eb-8b91-aefdb6cea484.png">

The fork Lightbox works. We should copy what that uses

Reply:
I reverted. I have found that box not always to work. I happen to think its not even worth fixing